spec/features/monitor/create_account_spec.rb

```diff
@@ -11,7 +11,7 @@
     context 'not staging' do
       before { monitor.filter_unless('STAGING') }
 
-      xit 'creates new account with SMS option for 2FA' do
+      skip 'creates new account with SMS option for 2FA' do
```

Comment:
why the switch from `xit` to `skip`? are these the same?

Reply:
Hm, I meant to reset this, and did `git checkout main -- spec/features`. I'm guessing the branch point is out of sync with main, hence why it appears there's changes.

Reply:
Looks like this was cleared up in the rebase I just pushed.